projects/Push-Based-Custom-Ceiling-Partial-Withdraw-for-EIP7251-MaxEB.md

```diff
@@ -0,0 +1,98 @@
+
+        1.  Any balance above 32 ETH earned through rewards does not actually contribute to principal, or increase the weight of this validator on the network, and is thus automatically withdrawn as a reward payment every few days.
+        2.  Aside from providing a withdrawal address one time, these rewards do not require any action from the validator operator. This is all initiated on the consensus layer, thus no gas (transaction fee) is required at any step.
+4.  After [EIP-7251](https://eips.ethereum.org/EIPS/eip-7251), MaxEB is increased from `32 ETH` to `2048 ETH`. It means there is a space for custom ceiling between `32 ETH` to `2048 ETH` which is defined by `MIN_ACTIVATION_BALANCE` and `MAX_EFFECTIVE_BALANCE`. 
+    1.  Be more specific, any excess balance beyond the custom ceiling should enjoy the same mechanism as partial withdraw beyond MaxEB. 
+    2.  Custom ceiling is not an issue [EIP-7251]((https://eips.ethereum.org/EIPS/eip-7251)), as both `both MAX_EFFECTIVE_BALANCE` and `MIN_ACTIVATION_BALANCE` equal to `32 ETH`. 
+5.  What is the impact if we do not implement this custom ceiling partial wirthdrawal along/after [EIP-7251]((https://eips.ethereum.org/EIPS/eip-7251)):
+    1.  Validators has to exit in order to use the stakes and rewards. 
+    2.  It causes more other partial withdrawal queue on Execution Layer, and cost more gas fee to the stakers.  
+6.  What is the benefti if we implement this feature:
+    1.  This feature benefits to solo stakers to improve their stake financial efficiency. 
+    2.  This feature benefits to institutional stakers to manage and control their staking strategy and operation.
+    3.  This feature benefits to stakers to reblance stake across node operators, to reduce the risk of lock-in by specific providers. 
+7.  The community discussed heavily on the timeline of this feature nearly in 5 out of 7 EIP-7251 breakout calls [1](https://hackmd.io/@wmoBhF17RAOH2NZ5bNXJVg/S1U86pzgR) [3](https://hackmd.io/@philknows/BJCaLJf1A#Custom-celings-To-be-continued-in-next-meeting) [4](https://hackmd.io/@philknows/Sy2kQAq1C?#Custom-Ceilings) [5](https://hackmd.io/YNy6vhDoQ8Ki6DQNv8tsWA#:~:text=In%20CALL%20%235%2C-,https%3A//hackmd.io/%40philknows/S1JbLXmlA%23Custom%2DCeilings,-Custom%20ceilings%20are) [6](https://hackmd.io/@philknows/Hywht12eR#Custom-Ceilings). And the conclusion is: 
+    1.  This feature should be implemented. 
+    2.  This feature has complexity, and the solid solution is not yet there. 
+    3.  Due to the tight timeline for Electra upgrade, it should be push into another EIP with a solid approach. 
+8.  This project target on this goal to design and implement a solid solution. 
+
+![whiteboard_exported_image (49)](https://hackmd.io/_uploads/BJDPl8BKC.png)
+
+## Specification
```

Comment:
Some text above, like benefits of the proposal should be rather in motivation. Now specs section is empty, it should describe exactly what are you implementing, link your repo/pr/specification..